app/routers/chat_stream.py

```
@@ -2,61 +2,74 @@
-        persist_directory=settings.CHROMA_DB_URI,
-        embedding_function=_embeddings,
-        collection_name="nebula_html",
-        client_settings=chromadb.config.Settings(
-            is_persistent=True,
-            persist_directory=settings.CHROMA_DB_URI,
-            anonymized_telemetry=False
-        )
-    )
-    logger.info("✅ 벡터DB 초기화 완료")
-    return _vectordb
+def _get_vectordb():
```

Pylint (3.3.7)

[refactor] Useless return at end of function or method

(R1711)